packages/api-metadata/src/extrinsics/fromMetadata/createUnchecked.ts

```diff
@@ -33,7 +33,7 @@ export default function createDescriptor (
       `Extrinsic ${section}.${funcName} expects ${expectedArgs.length.valueOf()} arguments, got ${args.length}.`
     );
 
-    return new (ClassOf('Method'))({
+    return createType('Method', {
```

Comment:
Because of these. There are also a places where this affects apps (and extension), so it is really worth-while having this work properly and without surprises.